src/ui/patchSettings.tsx

```diff
@@ -1,16 +1,16 @@
 import { sha } from "aliucord-version";
```

Comment:
can we move this file under somewhere else
like under src/patches 
\+ the themer/patcheTheme.tsx and then themerInit can go in root by itself